src/RunTests/build/Microsoft.Build.RunVSTest.props

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  
+  Licensed under the MIT license.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <NugetPath Condition="$(NugetPath)==''">$(NuGetPackageRoot)</NugetPath>
+    <NugetPath Condition="$(NugetPath)=='' And $(NUGET_PACKAGES)!=''">$(NUGET_PACKAGES)</NugetPath>
+    <SkipExecution Condition="$(SkipExecution)==''">false</SkipExecution>
```

Comment:
`SkipExecution` is pretty vague. `SkipTestExecution` maybe?

Also, why is this needed? I'm not sure I get the scenario of `msbuild /t:Build;Test /p:SkipExecution=true`